include/pq.h

```diff
@@ -31,9 +31,9 @@ class FixedChunkPQTable
     virtual ~FixedChunkPQTable();
 
 #ifdef EXEC_ENV_OLS
-    void load_pq_centroid_bin(MemoryMappedFiles &files, const char *pq_table_file, size_t num_chunks);
+    uint64_t load_pq_centroid_bin(MemoryMappedFiles &files, const char *pq_table_file, size_t num_chunks);
```

Comment:
nit: Consistent interface might be preferred here e.g. pq_table.get_memory_in_bytes()